src/battle_message.c

```diff
@@ -1720,7 +1721,31 @@ void BufferStringBattle(u16 stringId)
             StringCopy(gBattleTextBuff2, gMoveNames[sBattleMsgDataPtr->currentMove]);
 
         ChooseTypeOfMoveUsedString(gBattleTextBuff2);
-        stringPtr = sText_AttackerUsedX;
+        stringPtr = sText_AttackerUsedX; //Start looking here
+
+
+        //Victory Music Logic - CSR Drill Dozer
+        if(VarGet(VAR_TEMP_START_EVENT_BATTLE) > 0 && !FlagGet(FLAG_SYS_CSR_VICTORY)) {
+            switch(VarGet(VAR_TEMP_START_EVENT_BATTLE)) {
+                case 1:
+                    if(sBattleMsgDataPtr->currentMove == MOVE_ENDEAVOR) {
+                        BattleStopLowHpSound();
+                        RunScriptImmediately(FadeSongAndPlayVictory); //MUS_CSR_DRILL_DOZER
+                        FlagSet(FLAG_SYS_CSR_VICTORY);
+                    }
+                        
+                case 2:
+                    if(sBattleMsgDataPtr->currentMove == MOVE_CONFUSION)
+                        RunScriptImmediately(FadeSongAndPlayVictory); //MUS_CSR_DRILL_DOZER
```

Comment:
Would you want to cut out the low HP sound in these other situations as well?

Reply:
Yup - I was just editing that one line while testing, but I forgot to extend it ^^;

Admittedly it's probably better practice to have a dedicated function for all the "Victory" moments, but hey